.registryrc

```diff
@@ -1 +1 @@
-bde63f7c32e8d169d7e3163b14b5bb25bd3d5042
+95ba65182fc4de29228b1888977c5aca7bf1e3f2
```

Comment:
will update to a main commit after https://github.com/hyperlane-xyz/hyperlane-registry/pull/429 merged